drivers/pwm/pwm_mcux_sctimer.c

```diff
@@ -20,6 +20,9 @@ LOG_MODULE_REGISTER(pwm_mcux_sctimer, CONFIG_PWM_LOG_LEVEL);
 
 #define CHANNEL_COUNT FSL_FEATURE_SCT_NUMBER_OF_OUTPUTS
 
+/* Constant identifying that no event number has been set */
+#define EVENT_NOT_SET (FSL_FEATURE_SCT_NUMBER_OF_EVENTS + 1U)
```

Comment:
is there an event number 0? if so, maybe you don't need to define this

Reply:
Yes, the first event number assigned by the MCUX SDK is 0. Additionally, the hardware itself has EV_0_xxx registers for event 0.

Reply:
so then isnt FSL_FEATURE_SCT_NUMBER_OF_EVENTS itself a valid constant for this purpose

Reply:
Yeah- it should be. I'll use that, thanks